.github/workflows/release.yml

```diff
@@ -138,3 +138,18 @@ jobs:
         tag: ${{ github.ref }}
         file: /tmp/anycloud-macos.tar.gz
         asset_name: anycloud-macos.tar.gz
+
+    - name: Update homebrew
+      # TODO: should we create a new token or reuse the AUTOPUBLISH one?
+      run: |
+        alanversion=$(echo ${{github.ref}} | sed "s/refs\/tags\///")
+        [[ $alanversion != *'beta'* ]] && \
+        curl \
+        --fail \
+        -XPOST \
+        -H "Accept: application/vnd.github.everest-preview+json" \
+        -H "Content-Type: application/json" \
+        -H "authorization: token ${{secrets.AUTOPUBLISH}}" \
+        https://api.github.com/repos/alantech/homebrew-core/actions/workflows/update.yml/dispatches \
+        -d '{"ref": "auto-update", "inputs": {"alan_version": "'"$alanversion"'"}}' \
```

Comment:
Has this been tested? I'm pretty sure bash single quotes do not allow `$var` substitution. I also don't know what's going on with the `"'"$...` part.

I think this needs to be something like:

```bash
-d "{\"ref\": \"auto-update\", \"inputs\": {\"alan_version\": \"$alanversion\"}}" \
```

Reply:
Yes, I have tested both, release and prerelease and it works. This give me some war but how I have it it works

Reply:
I have to update the `ref` to `main`, though

Reply:
The `"'"$...` it is because it is a JSON, in that way I concatenate strings

Reply:
Oh! That's clever!